pypeit/spectrographs/keck_deimos.py

```diff
@@ -716,6 +718,27 @@ def bpm(self, filename, det, shape=None, msbias=None):
 
         return bpm_img
 
+    def get_lamps(self, fitstbl):
+        """
+        Extract the list of arc lamps used from header
+
+        Args:
+            fitstbl (`astropy.table.Table`_):
+                The table with the metadata for one or more arc frames.
+
+        Returns:
+            lamps (:obj:`list`) : List used arc lamps
+
+        """
+        # read and save the arc lamp in each frame
+        lamp_list = np.array([])
+        for col in fitstbl:
+                lamp_list = np.append(lamp_list, col['lampstat01'].split(' '))
+        # make it in Pypeit format
+        lamp_list = [lamp + 'I' for lamp in np.unique(lamp_list)]
+
+        return lamp_list
```

Comment:
Sorry to keep nit-picking, but you can make this whole function a one-liner:

```
return [f'{x}I' for x in np.unique(np.concatenate([l.split() for l in tbl['lampstat01']]))]
```

Reply:
Thanks! I made this change.